conf/st2.conf

```diff
@@ -27,6 +27,9 @@ enable = False
 mode = proxy
 logging = st2auth/conf/logging.conf
 
+# Base URL to the API endpoint excluding the version (e.g. http://myhost.net:9101/)
+api_url =
```

Comment:
It doesn't break auth, the header we return just won't have any value.

The thing with using localhost and running st2 + st2web on the same host is that it only really works in vagrant / vm scenario. st2web is a client side app which means it will only work if the user which is using web ui is also on the same host as st2 and st2web,